app/views/records/edit_fields/_qualification_level.html.erb

```diff
@@ -1,8 +1,14 @@
 <% qualification_level_service = HykuAddons::QualificationLevelService.new %>
 <% selected =  f.object[:qualification_level].empty? ? nil :  f.object[:qualification_level] %>
```

Comment:
Not sure if its weird formatting, but it looks like you've got double spaces in weird places, can you check?